deploy/common/k8sbased.sh

```diff
@@ -363,13 +363,25 @@ function launch_central {
         )
       fi
 
+      local helm_chart="$unzip_dir/chart"
+
+      if [[ -n "${CENTRAL_CHART_DIR_OVERRIDE}" ]]; then
+        helm_chart="${CENTRAL_CHART_DIR_OVERRIDE}"
+      fi
```

Comment:
I still (mentioned before [in this comment](https://github.com/stackrox/stackrox/pull/5795#discussion_r1296239866)) would suggest to inform user that we're leveraging the override chart

```suggestion
      if [[ -n "${CENTRAL_CHART_DIR_OVERRIDE}" ]]; then
        echo "Using override central helm chart from ${CENTRAL_CHART_DIR_OVERRIDE}"
        helm_chart="${CENTRAL_CHART_DIR_OVERRIDE}"
      fi
```